lib/ext/heroku/command/addons.rb

```diff
@@ -203,7 +203,14 @@ def create
 
       action("Creating #{addon['name'].downcase}") {}
       action("Adding #{addon['name'].downcase} to #{app}") {}
-      action("Setting #{addon['config_vars'].join(', ')} and restarting #{app}") do
+
+      message = ''
+
+      unless addon['config_vars'].empty?
+        message ="Setting #{addon['config_vars'].join(', ')} and restarting #{app}"
+      end
+
+      action(message) do
```

Comment:
Let me take another stab at cleaning this up, now that I know that, and then we'll get it merged.